lib/dialects/postgres/query-generator.js

```diff
@@ -259,10 +259,10 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
     return query;
   }
 
-  removeColumnQuery(tableName, attributeName) {
+  removeColumnQuery(tableName, attributeName, options) {
     const quotedTableName = this.quoteTable(this.extractTableDetails(tableName));
     const quotedAttributeName = this.quoteIdentifier(attributeName);
-    return `ALTER TABLE ${quotedTableName} DROP COLUMN ${quotedAttributeName};`;
+    return `ALTER TABLE ${quotedTableName} DROP COLUMN${options && !options.mustExist ? ' IF EXISTS' : ''} ${quotedAttributeName};`;
```

Comment:
What do you think about moving this logic of `options.mustExist` to a const, and late it, just use it on your return